packages/lib/src/Avatar.wc.svelte

```diff
@@ -44,18 +34,27 @@
           class={twMerge(
             "animate-spin",
             size === "sm" && "w-7 h-7",
-            size === "md" && "w-10 h-10",
+            size === "md" && "w-12 h-12",
```

Comment:
Tested making the spinner a bit bigger so it matches the radius of the avatar. Not sure what's right.

Reply:
I just set a value for it to be inside the circle.

I say what you said: Not sure what's right ;)

I can spend some time doing CSS on all our components before we ship anything for real production

Reply:
I reverted this change for now.